src/mainProg.scala

```diff
@@ -0,0 +1,27 @@
+import Numbers.NumbersToEnglishWords
+
+/**
+  * Created by aliaa on 20/04/17.
+  */
+object mainProg {
+  def main(args: Array[String]): Unit = {
+    val C1 = new NumbersToEnglishWords
+    //for(i <- 549751219 to 999999999) {
+      //println(C1 convertToWords i)
+    //}
+
+
+    println(C1 convertToWords 549751219.5)//val C1 = new NumbersToEnglishWords
```

Comment:
We need to try
- 10
- 100
- 1000
- 1000000
- 9
- 99
- 999
- 9999
- 99999
- 999999
- 101
- 111
- 1001
- 1101
- 1111
- 1011
- ...